http-module/src/main/scala/org/mule/weave/v2/module/http/functions/utils/HttpClientRequestConverter.scala

```diff
@@ -47,6 +49,24 @@ class HttpClientRequestConverter(
       })
     })
 
+    val maybeCookieHeader = headers.get(HttpHeader.COOKIE_HEADER)
+    val cookiesValue = selectObject(request, COOKIES).getOrElse(ObjectSeq.empty)
+
+    // Just one Cookie header is allowed
+    if (maybeCookieHeader.isDefined && maybeCookieHeader.get.nonEmpty && !cookiesValue.isEmpty()) {
+      throw new WeaveRuntimeException(s"A duplicate `cookie` field was found. Just one cookie value is allowed, it can not be specified as a 'header' value and as a 'cookie' object for a single request.", location.location())
```

Comment:
We should specify the name of the cookie that is duplicated. And I will say to create a new Exception that extends from WeaveRuntimeException